packages/ui/components/pagination/Pagination.tsx

```diff
@@ -12,10 +12,8 @@ export interface PaginationProps {
   currentPage: number;
   pageSize: number;
   totalItems: number;
-  onPageChange: (page: number) => void;
   onPageSizeChange: (pageSize: number) => void;
-  onNext?: () => void;
-  onPrevious?: () => void;
+  onPageChange: (page: number) => void;
```

Comment:
Maybe I wasn't clear. I'd keep onNext and onPrevious as optional props. They can be still useful for some other cases (although not in DataTablePagiation)